### ISSUE TITLE
Unable to run example demo

### ISSUE THREAD

**Issue by alexandery** [Nordstrom/kafka-connect-lambda#23]
Hi,

Thanks for the plugin. Trying it out using your instructions on a local docker instance. However the connector isn't working and I'm seeing the following exception in kafka connect logs:

```com.amazonaws.SdkClientException: Unable to find a region via the region provider chain. Must provide an explicit region in the builder or setup environment to supply a region.```

I have provided configuration and Kafka Connect seems to have it applied (which I can confirm using ```curl.exe http://localhost:8083/connectors/example-lambda-connector/config```).

What's interesting is that if I restart the connector via REST (```curl.exe -XPOST http://localhost:8083/connectors/example-lambda-connector/restart```) the exception doesn't get printed again, but the connector isn't working anyway.

My configuration, which I submit to Connect is based on your example:

```
{
  "name": "example-lambda-connector",
  "config": {
    "tasks.max": "1",
    "connector.class": "com.nordstrom.kafka.connect.lambda.LambdaSinkConnector",
    "topics": "example-stream",
    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
    "value.converter": "org.apache.kafka.connect.storage.StringConverter",
    "aws.region": "us-east-2",
    "aws.lambda.function.arn": "arn:aws:lambda:us-east-2:00000000:function:example-function",
    "aws.lambda.invocation.timeout.ms": "60000",
    "aws.lambda.invocation.mode": "SYNC",
    "aws.lambda.batch.enabled": "false"
  }
}
```

Would appreciate any pointers that would help me here.

**Comment by dylanmei** [comment]
Thanks for your report. This appears to be a recent regression. 

I intend to address this soon; as a workaround you should be able to pass AWS_REGION=us-east-2 as an environment variable to your Kafka Connect Docker container.

**Comment by alexandery** [comment]
Thank you for getting back to me, @dylanmei . The workaround behaves as described so I'm able to actually see the plugin work.

**Comment by dylanmei** [comment]
This is resolved via #24 and available in release `v1.1.1`.
